### PR TITLE
Add sanity login step to local setup.

### PR DESCRIPTION
**Pull request by jtfairbank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,16 @@
...
+## Docs
+
+  * [Sanity Docs](https://www.sanity.io/docs/overview-introduction)
+
 ## Local development
 
 After cloning the project, run:
 
     npm ci
+    npm run-script setup
```
Comment (coderbyheart) [inline README.md]:
```suggestion
    npm run login
```

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0-development",
   "description": "Sanity Studio configuration for Masks for Humanity",
   "scripts": {
+    "setup": "sanity login",
```
Comment (coderbyheart) [inline package.json]:
```suggestion
    "login": "sanity login",
```

**Review by coderbyheart — CHANGES_REQUESTED** [review]
It's a login, so why not name it this way?

**Comment by jtfairbank** [comment]
@coderbyheart gooood point.  When I was setting it up I thought there may be additional steps I'd need to include, but that turned out to not be the case.  Thanks for the refactor!

**Pull request merged**